include/openssl/digest.h

```diff
@@ -303,20 +303,19 @@ OPENSSL_EXPORT int EVP_MD_nid(const EVP_MD *md);
 // This function was deprecated from OpenSSL in BoringSSL, but was brought back
 // to AWS-LC.
 //
-// If |pctx| is not NULL, |EVP_MD_CTX_set_pkey_ctx| sets |ctx|'s |EVP_PKEY_CTX|
-// reference to |pctx|. The |EVP_PKEY_CTX| object |pctx| needs to have been
-// initialised before associating it with |ctx|. The hash functions associated
-// to |ctx| and |pctx| must be equal. Once |EVP_MD_CTX_set_pkey_ctx| is called,
-// the callee is responsible for freeing |pctx|. Calling |EVP_MD_CTX_cleanup|
-// will not free |pctx|.
+// |EVP_MD_CTX_set_pkey_ctx| sets |ctx|'s |EVP_PKEY_CTX| reference to |pctx|.
+// The |EVP_PKEY_CTX| object |pctx| needs to have been initialised before
+// associating it with |ctx|. The hash functions associated to |ctx| and |pctx|
+// must be equal. Once |EVP_MD_CTX_set_pkey_ctx| is called, the callee is
+// responsible for freeing |pctx|. Calling |EVP_MD_CTX_cleanup| will not free
+// |pctx|.
 //
-// The callee is responsible for the memory associated to |ctx|'s |EVP_PKEY_CTX|
-// reference, until |EVP_MD_CTX_set_pkey_ctx| is called with |pctx| equal to
-// NULL.
+// The caller is responsible for the memory associated to |ctx|'s |EVP_PKEY_CTX|
+// reference, until |EVP_MD_CTX_set_pkey_ctx| is called again with |pctx| = NULL.
```

Comment:
Doesn't the first part here mean "The caller is responsible for the memory associated to |pctx|"?